test/test_flatsat_adcs.py

```diff
@@ -21,17 +21,13 @@
 
 import time
 import sys
-from testLib import testLib as test
 from os import path
-sys.path.append("./src")
-from groundStation import groundStation
 
-import numpy as np
+from tester import Tester
+from eps.expected_eps_hk import expected_EPS_HK
+from test_full_hk import testSystemWideHK
 
-opts = groundStation.options()
-gs = groundStation.groundStation(opts.getOptions())
-
-test = test() #call to initialize local test class
+tester = Tester() #call to initialize local test class
```

Comment:
This variable name is a little confusing with line 26

Reply:
Yeah, I had a feeling. I changed the file name